docs/.vuepress/atomicDEX-sidebar.js

```diff
@@ -314,7 +314,14 @@ let atomicDEXsidebar = {
           "/basic-docs/atomicdex-api-20-dev/trade_preimage.md",
           "trade_preimage",
         ],
-        ["/basic-docs/atomicdex-api-20-dev/withdraw.md", "withdraw"],
+        [
+          "/basic-docs/atomicdex-api-20-dev/get_raw_transaction.md",
+          "get_raw_transaction",
```

Comment:
Could you please alphabetically reorder all methods?

Reply:
I'll add this as an issue and submit a separate PR to implement aplhabetic order s doing it in this PR may add further conflicts with other method PRs yet to merge